main/zoe/guide/contract-installation-and-instance.md

```diff
@@ -0,0 +1,98 @@
+# Contract installation and instance
+
+Zoe has two methods to "create a contract": 
+- `zoe.install(code, moduleFormat)` which returns an `installationHandle`
```

Comment:
Mark suggests always using `E`, as in `E(zoe).install`, which is probably the right way to go, but we will need to have an aside to an explanation of what `E` is.